src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFParagraph.java

```diff
@@ -402,6 +402,24 @@ public BigInteger getNumStartOverride() {
         }
         return null;
     }
+    
+    /**
+     * Indicates whether this paragraph should be kept on the same page as the next one.
+     */
+    public boolean isKeepNext() {
+        if (getCTP() != null && getCTP().getPPr() != null) {
```

Comment:
does this compile? seems to me, this has no return if the if-condition fails

Reply:
I've corrected this and added a test. Better?